app/jobs/export_notebook_job.rb

```diff
@@ -69,9 +76,12 @@ def set_transformation_matrix(pdf, filename)
       if token.is_a?(PDF::Reader::Token) && PDF::Reader::PagesStrategy::OPERATORS.key?(token)
         operator = PDF::Reader::PagesStrategy::OPERATORS[token]
         if operator == :concatenate_matrix
+          params[0] = (params[0] * 1.1875)
+          params[3] = (params[3] * 1.1875)
+          params[5] = (params[5].to_f * (4.0 / 3.0)).round
```

Comment:
I am unsure exactly of the reason behind these factors but they work so I'm happy. Definitely needs additional testing on a wider array of test PDFs